src/controllers/common.go

```diff
@@ -17,3 +17,11 @@ func convertMapToKeyValue(values map[string]string) []types.KeyValue {
 	}
 	return kvList
 }
+
+func convertKeyValueToByteData(kvList []types.KeyValue) map[string][]byte {
+	values := map[string][]byte{}
```

Comment:
I`d suggest you call differently maybe  something like parsedData because its the  a bit confusing with all the key values already in the function